app/src/main/res/layout/activity_send_sms_invitation.xml

```diff
@@ -44,13 +44,37 @@
         android:autofillHints=""
         tools:ignore="HardcodedText,TextFields" />
 
-    <Button
-        android:id="@+id/send"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="send sms to invite"
-        android:layout_gravity="center_horizontal"
-        tools:ignore="HardcodedText" />
+    <LinearLayout
+        android:layout_below="@+id/text_view_schedule_response_confirmation"
```

Comment:
Android lint is complaining: ObsoleteLayoutParam: Invalid layout param in a `LinearLayout`: `layout_below`

Reply:
ok, it' s removed